torch/_dynamo/variables/functions.py

```diff
@@ -1194,7 +1195,7 @@ def call_function(
                 if self.value.__module__ in known_python_builtin_modules:
```

Comment:
Missed a `self.value.__qualname__` on L1187.

Reply:
Good catch (redid the search and didn't spot any more, so hopefully this is the last one).